tests/ExecClientTests.cs

```diff
@@ -0,0 +1,41 @@
+using k8s.tests.Mock;
+using System;
+using System.Collections.ObjectModel;
+using System.Text;
+using System.Threading;
+using Xunit;
+
+namespace k8s.tests
+{
+    public class ExecClientTests
+    {
+        [Fact]
+        public void ReadWriteTest()
```

Comment:
cloud you please align with naming style only `verb` 
like https://github.com/kubernetes-client/csharp/pull/51